### ISSUE TITLE
Toggle to Start or Stop all jobs together

### ISSUE THREAD

**Issue by cee-mnitin** [dagster-io/dagster#8343]
## Use Case

There are number of times when we need to stop all jobs in a pipeline to be stopped immediately. In such cases, we have to go to each toggle one by one and start/stop it. A single toggle to toggle all jobs will be useful

## Ideas of Implementation

The best solution would be a toggle switch on top of the Status>Sensors page. It would act similar to a select all button which will toggle all jobs together.

## Additional Info

---
#### Message from the maintainers:
Excited about this feature? Give it a :thumbsup:. We factor engagement into prioritization.

**Comment by rexledesma** [comment]
Spiritually similar to #8032.

**Comment by hellendag** [comment]
Agree, this sounds like the same issue as #8032, so I'm going to close this as a duplicate of that one. It is currently in the core backlog -- I don't think we can solve this solely within UI.
